agent/api/task/task.go

```diff
@@ -583,7 +583,7 @@ func (task *Task) addNamespaceSharingProvisioningDependency(cfg *config.Config)
 		if container.IsInternal() {
 			continue
 		}
-		container.BuildContainerDependency(NamespacePauseContainerName, apicontainerstatus.ContainerResourcesProvisioned, apicontainerstatus.ContainerPulled)
+		container.BuildContainerDependency(NamespacePauseContainerName, apicontainerstatus.ContainerRunning, apicontainerstatus.ContainerPulled)
```

Comment:
You will also need to change 
```
namespacePauseContainer := apicontainer.NewContainerWithSteadyState(apicontainerstatus.ContainerResourcesProvisioned)
``` 
to 
```
namespacePauseContainer := apicontainer.NewContainerWithSteadyState(apicontainerstatus.ContainerRunning)
```
Otherwise the Agent will call `provisionContainerResources` to setup CNI for namespace pause container.

Reply:
Good catch. Updated.